docs: docs (#27)
bluefireteam/mini_sprite: README.md

[license_link]: https://opensource.org/licenses/MIT
[very_good_analysis_badge]: https://img.shields.io/badge/style-very_good_analysis-B22C89.svg
[very_good_analysis_link]: https://pub.dev/packages/very_good_analysis

<img src="art/logo.png" width="250" />

## Project structure

> Mini Sprite is composed by several projects:

Reviewer: ```suggestion
Mini Sprite is composed of several projects:
```

### [Mini Sprite](packages/mini_sprite)

> The core of it contains the engine/platform agnostic core code to handle the format for the

Reviewer 2: ```suggestion
The core package contains the engine/platform agnostic code to handle the format for the
```

> The core of the it contains the engine/platform agnostic core code to handle the format for the

Reviewer: ```suggestion
The core of it contains the engine/platform agnostic core code to handle the format for the
```

sprites and maps.

### [Mini Sprite Editor](packages/mini_sprite_editor)

> The flutter application deployed at [https://minisprit.es](https://minisprit.es) which is used to design sprites and
> maps using Mini Sprite's format.

Reviewer: ```suggestion
The flutter application, deployed at [https://minisprit.es](https://minisprit.es), which can be used to design sprites
and maps using the Mini Sprite format.
```

> ### [Flame Mini Sprite](packages/flame_mini_sprite)

Reviewer: Shouldn't the bridge library live in the Flame monorepo?

Author: It could! But I thought it is too soon for this to be a thing in the flame org.

Would love to move it though (but probably later with more time 😅 )

Provides bridge methods to make it easier to use Mini Sprite inside the
[flame game engine](https://flame-engine.org).

### [Mini Treasure Quest](packages/mini_treasure_quest)

An example game made with Flame and Mini Sprite.

### Get help

> Check each provide listed above to read the documentation of each package.

Reviewer: ```suggestion
Check each link provided above to read the documentation for each package.
```

> Join Bluefire's [discord server](https://discord.gg/pxrBmy4) to ask question to the team and the community.

Reviewer: ```suggestion
Join Bluefire's [discord server](https://discord.gg/pxrBmy4) to ask questions to the team and the community.
```
